frontend/add-client/services.component.tsx

```diff
@@ -82,6 +82,12 @@ const css = `
   margin-top: .8rem;
 }
 
+form {
```

Comment:
All css rules that use kremling should be scoped with the `&` character. See https://kremling.js.org/concepts/scoped-css.html

```suggestion
& form {
```